templates/include/top-nav-responsive.html

```diff
@@ -25,6 +25,7 @@
       <a href="https://bluebutton.cms.gov/developers/">Documentation</a>
       <a href="https://groups.google.com/forum/#!forum/Developer-group-for-cms-blue-button-api">Support</a>
       {% if 'apps.testclient' in settings.INSTALLED_APPS %}
+      <a href="{% url 'bluebutton_openapi' %}" target='_blank'>Open API</a>
```

Comment:
@micahtaylor Just curious on your thoughts w/ adding this to the top bar.

Reply:
Fine by me.

Reply:
top bar link has many benefits, or users won't see it for a long time

Reply:
I did some minor adjustment on A/C for that, thanks!